sdk/opendp/smartnoise/_ast/ast.py

```diff
@@ -67,16 +67,17 @@ def load_symbols(self, metadata):
         self.m_symbols = symbols
         self.m_sym_dict = {}
 
-        tables = []
         for name, symbol in self.m_symbols:
-            for table in symbol.find_nodes(TableColumn):
-                tables.append(table)
             if name == "???":
                 continue
             if name in self.m_sym_dict:
                 raise ValueError("SELECT has duplicate column names: " + name)
             self.m_sym_dict[name] = symbol
 
+        tables = []
+        for t in self.find_nodes(Table):
+            tables.append(t.m_symbols[0][1])
```

Comment:
I do not understand why `row_privacy` is loaded in `TableColumn` and not in `Table` since in `CollectionMetadata`, `row_privacy` is set for the table and not the line. 

Maybe I am missing something, but this change should fix https://github.com/opendp/smartnoise-sdk/issues/380 in the case were the tables are not empty. Should I raise an error otherwise ?